ProcessMaker/Models/FormalExpression.php

```diff
@@ -64,7 +80,7 @@ private function evaluate(array $data)
         $encoder = isset(self::languages[$language][1]) ? self::languages[$language][1] : null;
         try {
             $values = $encoder ? $this->$encoder($data) : $data;
-            return $this->$evaluator->evaluate($this->getBody(), $values);
+            return $this->$evaluator->evaluate($body, $values);
```

Comment:
@marcoAntonioNina can we add a test for this? I know there isn't a test file yet but it should be simple to create one for FormalExpression in tests/Model